docs/api_wiki/sonichost_methods/get_vlan_brief.md

```diff
@@ -0,0 +1,49 @@
+# get_vlan_brief
+
+- [Overview](#overview)
+- [Examples](#examples)
+- [Arguments](#arguments)
+- [Expected Output](#expected-output)
+- [Potential Exception](#potential-exception)
+
+## Overview
+
+Read vlan brief information from running config.
+
+## Examples
+
+```python
+def test_fun(duthosts, rand_one_dut_hostname):
+    duthost = duthosts[rand_one_dut_hostname]
+    vlan_brief = duthost.get_vlan_brief()
+```
+
+## Arguments
+
+None
+
+## Expected Output
+
+Returns an dict, key is vlan name and value is brief information.
+
+Example output:
+
+```
+{
+    "Vlan1000": {
+        "interface_ipv4": [],
```

Comment:
Can you add a sample IP address here. Also in the comments of your code.

Reply:
Ok, updated